src/core/raps/unlockAndCrosschainSwap.ts

```diff
@@ -135,6 +137,13 @@ export const createUnlockAndCrosschainSwapRap = async (
       assetToSell.address?.toLowerCase() as keyof PermitSupportedTokenList
     ];
 
+  if (claimHash) {
+    const claim = createNewAction('claim', {
+      claimHash: claimHash,
+    });
+    actions = actions.concat(claim);
+  }
```

Comment:
maybe i'm lost but why don't we have another rap for this?

needing a claimHash feels like you already send the tx and raps are built to execute the tx

Reply:
some random guy stepped in and took care of this